doc/array_context.rst

```diff
@@ -20,7 +27,7 @@ Lazy/Deferred evaluation array context based on :mod:`pytato`
 .. automodule:: arraycontext.impl.pytato
 
 
-Array context :mod:`jax.numpy`
+Array context based on :mod:`jax.numpy`
 -------------------------------------------------------------
```

Comment:
```suggestion
---------------------------------------
```

Reply:
Nice catch! Fixed in https://github.com/inducer/arraycontext/compare/b12875d4eac2b19d71f4a8495fc2ff136ce77dad..2983116866fe389e7b923cada1b79715b69bf4b1.

Reply:
Thanks!